internal/kms/kms/cluster/cluster.go

```diff
@@ -20,8 +20,8 @@ type KMS struct {
 }
 
 // New creates a new ClusterKMS.
-func New(salt []byte) *KMS {
-	return &KMS{salt: salt}
+func New(key []byte, salt []byte) *KMS {
+	return &KMS{masterKey: key, salt: salt}
 }
```

Comment:
```suggestion
func New(key []byte, salt []byte) (*KMS, error) {
	if len(key) = 0 || len(salt) == 0 {
		return errors.New("missing salt / master key")
	}
	return &KMS{masterKey: key, salt: salt}, nil
}
```
With the future removal of `CreateKEK`, we should ensure `New` creates a valid configuration